pipeline.sh

```diff
@@ -8,6 +8,8 @@ OUTPUT=""
 FORCE=""
 KEEP=""
 LOG_FILE="pipeline.log"
+ASSEMBLY=GRCh37
+PARALLEL_THREADS=4
```

Comment:
since threads and forks are a different thing, CPU_CORES would be a better variable name in my opinion.